Sources/AudioKit/Internals/Utilities/AudioKitHelpers.swift

```diff
@@ -294,3 +294,39 @@ extension String {
             .capitalized // If input is in llamaCase
     }
 }
+
+extension Double {
+    /// Map the value to a new range
+    /// Return a value on [from.lowerBound,from.upperBound] to a [to.lowerBound, to.upperBound] range
+    ///
+    /// - Parameters:
+    ///   - to: Desired range (Default: 0...1.0)
+    ///   - from: Source range (Default: 0...1.0)
+    public func mapped(to: ClosedRange<Double> = 0...1.0, from: ClosedRange<Double> = 0...1.0) -> Double {
+        return ((self - from.lowerBound) / (from.upperBound - from.lowerBound)) * (to.upperBound - to.lowerBound) + to.lowerBound
```

Comment:
To make this line read a bit better,  change the method signature to rename to and from, maybe like:

` mapped(to target: ClosedRange<Double> = 0...1.0, from source: ClosedRange<Double> = 0...1.0)`

Also, I'm wondering if it would like nicer as from-to:

` mapped(from source: ClosedRange<Double> = 0...1.0, to target: ClosedRange<Double> = 0...1.0)`

Reply:
I was debating switching the order, too. I like from-to better. Good stuff